src/components/command-bar/index.tsx

```diff
@@ -45,21 +79,37 @@ const CommandBarProvider = ({ children }: CommandBarProviderProps) => {
 		return () => {
 			document.removeEventListener('keydown', handleKeyDown)
 		}
-	}, [])
+	}, [toggleIsOpen])
 
 	return (
-		<CommandBarContext.Provider value={{ isOpen, setIsOpen }}>
+		<CommandBarContext.Provider
+			value={{ ...state, setCurrentCommand, toggleIsOpen }}
+		>
```

Comment:
future 💭: we'll want to `useMemo` the context value here to minimize re-renders on components which access the context value.

Reply:
Good call! I can totally make that update in [Make CommandBarDialog track current tags in its state](https://app.asana.com/0/1201147515801429/1202932738893665/f)